dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/DiscretePI.mo

```diff
@@ -16,7 +16,6 @@ block DiscretePI "Proportional integrator with discrete input"
   import Dynawo.Types;
   import Modelica;
 
-  parameter Real Gain "Control gain";
```

Comment:
If we remove the P part, then the model (and its preassembled) should be renamed DiscreteIntegrator